src/main/java/com/smartbear/readyapi/plugin/git/ReadyApiGitIntegration.java

```diff
@@ -152,14 +152,22 @@ public CommitResult commit(Collection<VcsUpdate> vcsUpdates, String commitMessag
         final WsdlProject project = update.getProject();
         final Git gitObject = gitCommandHelper.createGitObject(project.getPath());
 
-        final boolean successfulUpdate = gitCommandHelper.commitAndPushUpdates(vcsUpdates, commitMessage, gitObject);
-
-        CommitResult result = successfulUpdate ? new CommitResult(SUCCESSFUL, "Commit was successful") :
-                new CommitResult(FAILED, "Commit Failed");
+        final GitCommandHelper.CommitStatus successfulUpdate = gitCommandHelper.commitAndPushUpdates(vcsUpdates, commitMessage, gitObject);
 
         gitObject.getRepository().close();
 
-        return result;
+        switch (successfulUpdate) {
+            case OK:
+                return new CommitResult(SUCCESSFUL, "Commit was successful");
+            case FAILED_PUSH:
+                // here I use FAILED instead of PARTIAL,
+                // because PARTIAL means that part of changes is in the r e m o t e repository,
+                // but in this case all changes are rejected
+                return new CommitResult(FAILED, "Local commit was successful, but push to remote was failed. Try to push the changes manually.");
```

Comment:
I think this message must tell user that he should use external tools.

Reply:
The message was changed